frontend/src/components/common/Resource/Resource.tsx

```diff
@@ -384,6 +385,82 @@ export function ConditionsTable(props: ConditionsTableProps) {
   );
 }
 
+interface VolumeMountsProps {
+  mounts?:
+    | {
+        mountPath: string;
+        name: string;
+        readOnly: boolean;
+      }[];
+}
+
+function VolumeMounts(props: VolumeMountsProps) {
+  const { mounts } = props;
+  if (!mounts) {
+    return null;
+  }
+  const classes = useMetadataDisplayStyles({});
+  return (
+    <Grid container direction="column" spacing={2}>
+      {mounts?.map((item, index) => (
+        <Grid item key={`${item.name}__${index}`}>
+          <Box display="flex">
+            <Typography className={classes.metadataValueLabel}>{item.mountPath}</Typography>
+            <Box px={2}>from</Box>
+            <Typography className={classes.metadataValueLabel}>{item.name}</Typography>
+            <Typography className={classes.metadataValueLabel} display="block">
+              <span>{item.readOnly ? '(readOnly)' : '(readWrite)'}</span>
+            </Typography>
```

Comment:
This needs to be a single paragraph, otherwise it looks okayish visually but will be a mess for screenreaders.
I will add a fixup commit for this.

Reply:
And actually it looks a bit overcrowded when there are many volumes.
Maybe we should use a table for this.

![image](https://user-images.githubusercontent.com/1029635/146222209-5e83d326-e33b-41ee-891e-a3474b88a6f1.png)

Reply:
![Screenshot from 2021-12-20 16-53-46](https://user-images.githubusercontent.com/30126128/146760095-f7b7aad4-c675-4411-a15e-ade9fc86c58b.png)